adonthell (0.3.3-2.kon.1) local; urgency=low

  * Local version!
  * src/str_hash.h: Specialize __gnu_cxx::hash, not std::hash.
  * src/storage.h, src/win_manager.h: Use __gnu_cxx::hash_map.
    (Closes: #177307)

 -- Kalle Olavi Niemitalo <kon@iki.fi>  Sat, 18 Jan 2003 19:44:33 +0200

diff -ru adonthell_0.3.3-2/src/str_hash.h adonthell_0.3.3-2.kon.1/src/str_hash.h
--- adonthell_0.3.3-2/src/str_hash.h	2003-01-18 19:38:33.000000000 +0200
+++ adonthell_0.3.3-2.kon.1/src/str_hash.h	2003-01-18 19:23:40.000000000 +0200
@@ -30,13 +30,13 @@
 #endif
 #include <string>
 
-namespace std 
+namespace __gnu_cxx 
 {
 	
-template<> struct hash<string> {
-    size_t operator()(const string& s) const {
+template<> struct hash<std::string> {
+    size_t operator()(const std::string& s) const {
 	return __stl_hash_string(s.c_str());
 }};
 
-}; // namespace std
+}; // namespace __gnu_cxx
 #endif
diff -ru adonthell_0.3.3-2/src/storage.h adonthell_0.3.3-2.kon.1/src/storage.h
--- adonthell_0.3.3-2/src/storage.h	2002-09-30 18:05:41.000000000 +0300
+++ adonthell_0.3.3-2.kon.1/src/storage.h	2003-01-18 19:25:34.000000000 +0200
@@ -40,6 +40,7 @@
 
 #ifndef SWIG
 using namespace std; 
+using __gnu_cxx::hash_map;
 #endif
 
 
diff -ru adonthell_0.3.3-2/src/win_manager.h adonthell_0.3.3-2.kon.1/src/win_manager.h
--- adonthell_0.3.3-2/src/win_manager.h	2002-09-30 18:05:41.000000000 +0300
+++ adonthell_0.3.3-2.kon.1/src/win_manager.h	2003-01-18 19:24:53.000000000 +0200
@@ -35,6 +35,7 @@
 
 #ifndef SWIG
 using namespace std;   
+using __gnu_cxx::hash_map;
 #endif
 
 /** 
